aten/src/ATen/native/native_functions.yaml

```diff
@@ -1425,6 +1425,8 @@
 
 - func: zero_(Tensor self) -> Tensor
 
+
+
```

Comment:
nit: remove whitespace

Reply:
I thought the extra whitespace at couple different places in this file were used to create "boundaries". Should I follow the same convention?

Reply:
I think that's a valid point, extra space might not be a good convention to separate different functions, either does it applied at all places in the file. Maybe to use comments is a better way to do it...